.github/workflows/build-push-docker.yml

```diff
@@ -39,23 +42,42 @@ jobs:
       - name: Login to GAR
         run: |
           gcloud auth configure-docker us-docker.pkg.dev -q
+      
+      - name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Not exactly sure if the GH token provided has correct permissions.

Reply:
Same here regarding testing